Makefile

```diff
@@ -2,13 +2,14 @@
 # AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY
 # OR AWS_PROFILE
 
-NAME                       := sanctuary/api
+TAG                        := v0.1.0
+CONTAINER_NAME					   := sanctuary_api
```

Comment:
looks like some tabs vs spaces issues here.  It's up to you if that is worth correcting.